### PR TITLE
fix check_aggregate with sensu-server 1.x.x

### PR DESCRIPTION
**Pull request by nishiki**
See the PR #33

### CODE REVIEW

**Review by majormoses — APPROVED** [review]
Thanks for the contribution, can I get a changelog entry?

**Comment by nishiki** [comment]
It's done

File: CHANGELOG.md
Comment (majormoses) [inline CHANGELOG.md]: generally it's best to leave this to the maintainer who merges it to version it since there is no guarantee on merge order but in this case I am not gonna make you change it since I can get to it shortly.

Comment (nishiki) [reply CHANGELOG.md]: sorry, it's fixed

**Comment by majormoses** [comment]
released: https://rubygems.org/gems/sensu-plugins-sensu/versions/1.1.1

**Pull request merged**
